cmd/subctl/verify.go

```diff
@@ -142,6 +143,7 @@ func addVerifyFlags(cmd *cobra.Command) {
 		"namespace in which submariner is deployed")
 	cmd.Flags().StringVar(&verifyOnly, "only", strings.Join(getAllVerifyKeys(), ","), "comma separated verifications to be performed")
 	cmd.Flags().BoolVar(&disruptiveTests, "disruptive-tests", false, "enable disruptive verifications like gateway-failover")
+	cmd.Flags().BoolVar(&useSmallPackets, "use-small-packets", false, "use small packets in connectivity verification")
```

Comment:
instead of use-small-packets, what are your thoughts on supporting packet-size?

Reply:
👍 That was my initial thought as well...

Reply:
I assume that supporting packet-size means we should update shipyard code ( check [1] and [2] ) as currently, we send data-buffer (sized  16bytes uuid converted to string 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' ) multiple X  times. 

I personally don't think we should be that flexible, I assume 2 packet size , normal and small should do the work for us. 

Any special use cases that you think flexible packet size can help with? 
Let me know If you think supporting normal/small packet size isn't sufficient  


[1] 
https://github.com/submariner-io/shipyard/blob/devel/test/e2e/framework/network_pods.go#L106
[2]
https://github.com/submariner-io/shipyard/blob/devel/test/e2e/framework/network_pods.go#L269

Reply:
I guess if there's a case where hard-coded normal/small packet sizes don't suffice?

Reply:
OK, I changed the flag to packet size